Twig/Extension/PostMetaExtension.php

```diff
@@ -49,6 +49,7 @@ public function getFunctions()
     {
         return array(
             new \Twig_SimpleFunction('wp_get_post_meta', array($this, 'getPostMeta')),
+            new \Twig_SimpleFunction('wp_get_image_url_from_id', array($this, 'getImageUrlFromId')),
```

Comment:
It will be possible to create a proxy twig helper ;) {{ wp('get_post_meta', $postId) }}